.github/workflows/runner.yml

```diff
@@ -0,0 +1,33 @@
+name: '[EXPERIMENTAL] Inspect environment'
+
+on:
+  workflow_call:
+    outputs:
+      config:
+        description: "The runner's configuration"
+        value: ${{ jobs.choose.outputs.config }}
+
+jobs:
+  choose:
+    runs-on: ubuntu-latest
+    outputs:
+      config: ${{ steps.config.outputs.result }}
+    steps:
+      - uses: actions/github-script@v6
+        id: config
+        with:
+          script: |
+            if (`${context.repo.owner}/${context.repo.repo}` === 'ipfs/kubo') {
+              return {
+                labels: ['self-hosted', 'linux', 'x64', 'kubo'],
+                parallel: 10,
+                aws: true
```

Comment:
:thought_balloon: 
@galargh @BigLep 
After we merge this PR, how long we need to run old setup on circleci and github actions with self-hosted worker on AWS in parallel to be able to compare costs? 1-2 months?

Reply:
If I can get a ballpark estimate on the cost that will help me determine whether this is even worth paying attention.  Making our setup easier to support/maintain and let external developers more easily fork and run CI is worth a lot.

Reply:
Really rough estimate here and I totally might have missed something but I think it's gonna work out to something like this:
- GitHub Actions: ~$99 / month (30 days * $2 for ALB, Gateway, Lambda, etc. + [261 runs in the past 30 days](https://app.circleci.com/insights/github/ipfs/kubo/workflows/test/overview?branch=b1d8acd4-8701-4c76-8a87-7ff2320aa95f) * $0.15 for EC2, EBS, S3, etc.)
- CircleCI: ~$468 / month ([780427 credits used in the past 30 days](https://app.circleci.com/insights/github/ipfs/kubo/workflows/test/overview?branch=b1d8acd4-8701-4c76-8a87-7ff2320aa95f) * [$0.0006 per credit](https://discuss.circleci.com/t/performance-plan-billing-update/39502))